components/scream/src/physics/spa/tests/CMakeLists.txt

```diff
@@ -15,5 +15,4 @@ CreateUnitTest(spa_one_to_one_remap_test "spa_one_to_one_remap_test.cpp" "${NEED
   MPI_RANKS 1 ${SCREAM_TEST_MAX_RANKS}
 )
 
-CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/spa_data_for_testing.nc
-               ${CMAKE_CURRENT_BINARY_DIR}/spa_data_for_testing.nc COPYONLY)
+GetInputFile(init/spa_data_for_testing.nc ${CMAKE_CURRENT_BINARY_DIR}/spa_data_for_testing.nc)
```

Comment:
I'm all for removing from our repo files needed for cime runs, and force a download (if not present locally). But this file is not for CIME stuff, only for internal testing. I can see it both ways, but my first thought would be "stuff for internal scream tests can live in the repo". It would make developing on local laptops/machines easier, no?

Reply:
This is related to your general comments above. This should not make it any harder to do local work; just pick a SCREAM_INPUT_ROOT, stick it in your cache file, and you should be good. The only exception to this would be if you do not currently have an internet connect and you are in needed of downloading some new files.